packages/cli-tools/bin/streamr-identity-generate.ts

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env node
+import '../src/logLevel'
+import { KeyPairIdentity } from '@streamr/sdk/dist/types/src/identity/KeyPairIdentity'
+import { IdentityMapping, validKeyTypeValues } from '@streamr/sdk'
+import { binaryToHex } from '@streamr/utils'
+import { createCommand, Options } from '../src/command'
+import { formEnumArgValueDescription, createFnParseEnum } from '../src/common'
+
+createCommand()
+    .description('generate a public/private key pair based on the --key-type option')
+    .requiredOption('--key-type [key-type]', `type of public/private key (${formEnumArgValueDescription(validKeyTypeValues)})`, 
+        createFnParseEnum('key-type', validKeyTypeValues))
+    .action(async (options: Options) => {
+        const config = IdentityMapping[options.keyType!] // required option
+        if (!config) {
+            console.error(`Error: Invalid key type. Must be one of: ${validKeyTypeValues.join(', ')}.`)
+        }
```

Comment:
This is not needed as `createFnParseEnum()` does the validation:

https://github.com/streamr-dev/network/blob/a4c6a0c3882b1fd0e8f4afaca2f6054bdb4117ff/packages/cli-tools/src/common.ts#L32-L33

```suggestion
```